package.json

```diff
@@ -17,7 +17,7 @@
     "build": "babel src -d lib --source-maps",
     "start": "babel src -w -d lib --source-maps",
     "clean": "rimraf lib",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build; git rev-parse HEAD > version.txt"
```

Comment:
mmhmm. well, suggestions for alternative solutions? 

we could have jenkins create `version.txt` before running `npm pack`, but that feels like it is building a different sort of package to normal.

Or we could have jenkins store a separate `matrix-js-sdk-version.txt` file alongside the tarball, which dependent builds then retrieve at the same time as retrieving the tarball.